commands/config.go

```diff
@@ -78,6 +178,69 @@ func configGetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func configSetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	config, response := c.GetConfig()
+	if response.Error != nil {
+		return response.Error
+	}
+
+	path, err := parseConfigPath(args[0])
+	if err != nil {
+		return err
+	}
+	if err := setConfigValue(path, config, args[1:]); err != nil {
+		return err
+	}
+	if _, res := c.UpdateConfig(config); res.Error != nil {
+		return res.Error
+	}
+
+	printer.Print("Value changed successfully")
+	return nil
+}
+
+func configResetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	confirmFlag, _ := cmd.Flags().GetBool("confirm")
+
+	if !confirmFlag && len(args) > 0 {
+		var confirmResetAll string
+		confirmationMsg := fmt.Sprintf(
+			"Are you sure you want to reset %s to their default value? (YES/NO): ",
+			args[0])
+		printer.Print(confirmationMsg)
+		fmt.Scanln(&confirmResetAll)
+		if confirmResetAll != "YES" {
+			printer.Print("Reset operation aborted")
+			return nil
+		}
+	}
+
+	defaultConfig := &model.Config{}
+	defaultConfig.SetDefaults()
+	config, response := c.GetConfig()
+	if response.Error != nil {
+		return response.Error
+	}
+
+	for _, arg := range args {
+		path, err := parseConfigPath(arg)
+		if err != nil {
+			return err
+		}
+		defaultValue, ok := getValue(path, *defaultConfig)
+		if !ok {
+			return errors.New("Invalid key")
+		}
+		resetConfigValue(path, config, defaultValue)
+	}
+	if _, res := c.UpdateConfig(config); res.Error != nil {
+		return res.Error
+	}
+
+	printer.Print("Value/s reset successfully")
+	return nil
+}
+
 func configShowCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	printer.SetSingle(true)
 	printer.SetFormat(printer.FORMAT_JSON)
```

Comment:
Is this right? Our help says default value for `--format` is plain but we're showing only JSON in this command

Reply:
In this specific case, the way to show the documentation is the same in both formats (a JSON entry) hence the fixed use of JSON.

We could potentially have another visualization for the "plain" mode if there is one that makes sense